src/dpp/sslclient.cpp

```diff
@@ -504,6 +504,15 @@ void ssl_client::read_loop()
 							case SSL_ERROR_NONE:
 								/* Data received, add it to the buffer */
 								if (r > 0) {
+									const std::string data(server_to_client_buffer, r);
+									const std::vector<std::string>& data_lines = utility::tokenize(data);
+									const std::string& http_reponse(data_lines[0]);
```

Comment:
don't use a reference, it is not valid after the function returns

Reply:
For both?

Reply:
yes just 
`const std::vector<std::string> data_lines =`
is fine